nurseconnect/views.py

```diff
@@ -1,15 +1,71 @@
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.models import User
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
 from django.utils.translation import get_language_from_request
-
-from molo.core.utils import get_locale_code
+from django.views.generic import FormView
+from django.views.generic import TemplateView
 from molo.core.models import ArticlePage
+from molo.core.utils import get_locale_code
+from molo.profiles import views
+from nurseconnect import forms
 from wagtail.wagtailsearch.models import Query
 
 
+class HomeView(TemplateView):
+    template_name = "core/main.html"
+
+    def render_to_response(self, context, **response_kwargs):
+        return render(self.request, self.template_name, context)
+
+
+class NurseConnectRegistrationView(views.RegistrationView):
+    form_class = forms.NurseConnectRegistrationForm
+
+    def form_valid(self, form):
+        username = form.cleaned_data["username"]
+        password = form.cleaned_data["password"]
+
+        user = User.objects.create_user(username=username, password=password)
+        user.profile.mobile_number = form.cleaned_data["mobile_number"]
```

Comment:
These are all still being set explicitly, it won't scale if the form grows larger.